adam-core/src/test/scala/org/bdgenomics/adam/converters/VariantContextConverterSuite.scala

```diff
@@ -231,11 +232,17 @@ class VariantContextConverterSuite extends FunSuite {
     val adamGT2 = adamVCs(1).genotypes.head
     assert(adamGT1.getAlleles.sameElements(List(GenotypeAllele.Alt, GenotypeAllele.OtherAlt)))
     assert(adamGT1.getAlternateReadDepth === 2)
-    assert(adamGT1.getGenotypeLikelihoods.sameElements(List(59, 0, 181)))
+    assert(adamGT1.getGenotypeLikelihoods
+      .map(f => f: scala.Float)
+      .map(PhredUtils.logProbabilityToPhred)
+      .sameElements(List(59, 0, 256)))
```

Comment:
Previously, since we were going Phred to Phred, we had enough precision to express this. Going Phred to log probabilities, we seem to clip around 1-1e13. Does anyone think this is a problem? I personally don't think it makes sense to worry much about the sensitivity of probabilities that are asymptotically close to 1.0.